File: DuDoong-Infrastructure/src/main/java/band/gosrock/infrastructure/outer/api/tossPayments/dto/response/PaymentsCard.java
package band.gosrock.infrastructure.outer.api.tossPayments.dto.response;


import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class PaymentsCard {
    private Long amount;
    private CardCode issuerCode;
    private CardCode acquirerCode;
    private String number;
    private Long installmentPlanMonths;
    private String approveNo;
    private Boolean useCardPoint;
    private String cardType;
    private String ownerType;
    private CardAcquireStatus acquireStatus;

    private Boolean isInterestFree;
    private String interestPayer;
}

File: DuDoong-Infrastructure/src/main/kotlin/band/gosrock/infrastructure/outer/api/tossPayments/dto/response/PaymentsCard.kt
package band.gosrock.infrastructure.outer.api.tossPayments.dto.response

class PaymentsCard {
    var amount: Long? = null
    var issuerCode: CardCode? = null
    var acquirerCode: CardCode? = null
    var number: String? = null
    var installmentPlanMonths: Long? = null
    var approveNo: String? = null
    var useCardPoint: Boolean? = null
    var cardType: String? = null
    var ownerType: String? = null
    var acquireStatus: CardAcquireStatus? = null
    var isInterestFree: Boolean? = null
    var interestPayer: String? = null
}
